packages/fleather/example/lib/main.dart

```diff
@@ -113,3 +141,41 @@ class _HomePageState extends State<HomePage> {
     }
   }
 }
+
+class ForceNewlineForInsertsAroundInlineImageRule extends InsertRule {
+  @override
+  Delta? apply(Delta document, int index, Object data) {
+    if (data is! String) return null;
+
+    final iter = DeltaIterator(document);
+    final previous = iter.skip(index);
+    final target = iter.next();
+    final cursorBeforeBlockEmbed = _isInlineImage(target.data);
```

Comment:
Although only in example, maybe rename to `cursorBeforeInlineEmbed`?

Reply:
Renamed